terraform/app.tf

```diff
@@ -126,6 +126,20 @@ resource "azurerm_linux_web_app" "aytp-app" {
     http2_enabled       = true
     minimum_tls_version = "1.2"
     health_check_path   = "/health"
+    ip_restriction = [{
+      name     = "FrontDoor"
+      action   = "Allow"
+      priority = 1
+      headers = [{
+        x_azure_fdid      = [local.infrastructure_secrets.FRONTDOOR_ID]
```

Comment:
Can we get the id via a data source instead? https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/cdn_frontdoor_profile#id

Reply:
OK it makes sense!